wiki/redirect.yaml

```
@@ -268,30 +268,44 @@
 "beatmap_submission_system":   "Submission"
 "changelog":                   "Glossary/Changelog"
 "changelogs":                  "Glossary/Changelog"
-"choke":                       "Glossary/Choke"
-"reverse_choke":               "Glossary/Choke"
+"choke":                       "Gameplay/Choke"
+"reverse_choke":               "Gameplay/Choke"
+"glossary/choke":              "Gameplay/Choke"
 "collab":                      "Beatmap/Beatmap_collaborations"
 "collaboration":               "Beatmap/Beatmap_collaborations"
 "combo":                       "Disambiguation/Combo"
 "combos":                      "Disambiguation/Combo"
 "combo_colour":                "Glossary/Combo_colour"
 "combo_color":                 "Glossary/Combo_colour"
-"combo_fire":                  "Glossary/Combo_fire"
-"combofire":                   "Glossary/Combo_fire"
-"combo_milestone":             "Glossary/Combo_milestone"
-"combo_multiplier_effect":     "Glossary/Combo_multiplier_effect"
-"combobreak":                  "Glossary/Combobreak"
-"combo_break":                 "Glossary/Combobreak"
-"comboburst":                  "Glossary/Comboburst"
-"combobursts":                 "Glossary/Comboburst"
-"combo_burst":                 "Glossary/Comboburst"
-"combo_bursts":                "Glossary/Comboburst"
+"combo_fire":                  "Gameplay/Combo_fire"
+"combofire":                   "Gameplay/Combo_fire"
+"glossary/combo_fire":         "Gameplay/Combo_fire"
+"combo_milestone":             "Gameplay/Combo_milestone"
+"glossary/combo_milestone":    "Gameplay/Combo_milestone"
+"combo_multiplier_effect":     "Gameplay/Combo_multiplier_effect"
+"glossary/Combo_multiplier_effect": "Gameplay/Combo_multiplier_effect"
```

Aren't all redirects supposed to be lowercase on the left? there seem to be some instances elsewhere in this file, but I'm not entirely sure if they might have been on purpose

```suggestion
"glossary/combo_multiplier_effect": "Gameplay/Combo_multiplier_effect"
```

---

case-insensitive on the left

---

ok nevermind it is lowercase only now :100: